config/environments/development.rb

```diff
@@ -5,6 +5,7 @@
   # every request. This slows down response time but is perfect for development
   # since you don't have to restart the web server when you make code changes.
   config.cache_classes = false
+  config.jbuilder.prettify = true
```

Comment:
To DRY out the code, moving it to application.rb surely will work since it's global for both development and production.